README.md

```diff
@@ -160,7 +160,35 @@ For `priceDrop` and `backInStock`
 
 ### `syncCart`
 
-Cart syncing is auto-enabled in this module. This API gets called automatically whenever the cart is updated.
+```javascript
+import { useCartState } from "frontend-checkout";
+import { processCart } from "@pushowl/shogun-frontend-sdk";
+
+const cart = useCartState();
+const cartId = cart.id;
+const cartItems = cart.items;
+
+React.useEffect(() => {
+  if (hasLoaded && cartItems && cartId) {
+    window.pushowl.trigger("syncCart", processCart({ cartItems, cartId }));
+  }
+}, [cartId, cartItems, hasLoaded]);
```

Comment:
where is `hasLoaded` suppose to come from?

Reply:
This is actually from `usePushOwl` hook. I have fixed the doc now now.